### PR TITLE
Resolves: 256 update doc of mor thermal building with python

### LINKED ISSUE #256: Update doc of mor thermal  building with python
Now we can add code and execute them in documentation, using feelpp/feelpp#2151 we can run the mor online code of thermal building

### PR DESCRIPTION
**Pull request by lberti**
- closes #256 

Add Python code for the online execution of the thermalbuilding model.

### CODE REVIEW

**Comment by lberti** [comment]
@prudhomm 
Locally I was not able to check if the notebook is correctly generated. 
I tested the code blocks, and they run. However, for some reason, the errors don't seem to be correctly computed in this example.

**Comment by prudhomm** [comment]
Which error type did you used?
Do you have the log of the offline code?
Have you tried to make it parabolic?
Can you plug external temperature from meteo data?

File: docs/mor/modules/ROOT/pages/thermalbuilding/index.adoc [hunk 1]
```diff
@@ -126,10 +128,50 @@ The table displays the various fixed and variables parameters of this test-case.
...
+| stem:[k_0] | Air conductivity     | stem:[1]
+| stem:[k_1] | Conductivity - internal walls | stem:[0.25]
+| stem:[k_2] | Conductivity - internal doors  | stem:[0.13]
+| stem:[\sigma]  | Heat transfer coefficient - front door | stem:[\frac{1.0}{0.06+\frac{0.06}{0.150}+\frac{0.1}{0.029}+0.14}]
+|=======================================================================
+
+
+== Running the case using a Jupyter notebook
+
+It is possible to download this page as a Jupyter notebook and run it in an environment that contains a local installation of {feelpp} and its Python wrappers. The offline creation of the reduced basis has already been performed, and an archive is downloaded from Girder. It contains the basis, the model and the configuration files that are necessary for the online simulation.
```
Comment (prudhomm) [inline docs/mor/modules/ROOT/pages/thermalbuilding/index.adoc]: Perhaps we should make this a partial to be used by the case docs?
- one for the jupyter Notebook
- one for the girder data
What do you think?

File: docs/mor/modules/ROOT/pages/thermalbuilding/index.adoc [hunk 2]
```diff
@@ -126,10 +128,50 @@ The table displays the various fixed and variables parameters of this test-case.
...
+import feelpp
+from feelpp.mor import *
+ms=feelpp.mor.MORModels("thermalbuilding.fpp")
+muspace = ms.parameterSpace()
+sampling = muspace.sampling()
+sampling.sample(4, "random")
+r=ms.run(sampling,{"N":10})
+----
+
+.Print the outputs and the associated parameter vector
```
Comment (prudhomm) [inline docs/mor/modules/ROOT/pages/thermalbuilding/index.adoc]: @lberti you you could create a panda data frame and use it to have a nice table?

**Review by prudhomm — APPROVED** [review]

**Comment by lberti** [comment]
**Error type**: residual scm
**Offline code**: the error estimation that is displayed during the offline part is of the order 1e-7, and when the fem solution is also computed, the error is of the order 1e-10; it's only in the online part that the error is not properly computed
**Parabolic**: no, it's just the stationary part, as it was implemented in the example `thermalbuilding.` I needed to document it to check what the model was like. I am working on the parabolic one
**External temperature**: the fifth parameter is the external temperature, involved in the computation of heat fluxes through the walls and the door. I did not add the possibility to read the external temperature from a meteo file.

File: docs/mor/modules/ROOT/pages/thermalbuilding/index.adoc (hunk 1 above)
Comment (lberti) [reply docs/mor/modules/ROOT/pages/thermalbuilding/index.adoc]: You mean to have two templated partials that can be reused for every `mor` case with an `fpp` archive, by specialising the commands to the given case?

**Comment by prudhomm** [comment]
Regarding the last point, i made a remark in the course notes regarding input signals over time at the end. We have to train with dirac signal at initial time.
Initial condition are already handled

File: docs/mor/modules/ROOT/pages/thermalbuilding/index.adoc (hunk 1 above)
Comment (lberti) [reply docs/mor/modules/ROOT/pages/thermalbuilding/index.adoc]: I have created two partials `girder_fpp.adoc` and `online_notebook.adoc`. Their variables are generic, and in each antora page it's necessary to specify 

- the girder path of the .fpp to be downloaded
- the name of the fpp file
- the list of time instants to be shown in the pandas dataframe

These variables are then used in the code contained in the partials

File: docs/mor/modules/ROOT/pages/thermalbuilding/index.adoc (hunk 2 above)
Comment (lberti) [reply docs/mor/modules/ROOT/pages/thermalbuilding/index.adoc]: I have created a Pandas data frame. The output of this test case is still weird though

**Review by prudhomm — COMMENTED** [review]
minor changes to use netlify

**Comment by prudhomm** [comment]
@lberti here is the page: https://deploy-preview-258--adoring-edison-f92f3c.netlify.app/mor/latest/thermalbuilding/

unfortunately since feelpp cannot be installed on netlify images we cannot check the results of the python code

**Comment by lberti** [comment]
@prudhomm In any case the Python code needs the developments contained in branch feelpp/2143-clean-mor to run the reduced model with the fpp archive.

**Pull request merged**